index.html

```diff
@@ -56,8 +58,16 @@
         /></a>
       </nav>
     </header>
+    <section class="mobile-menu">
+      <a href="#" class="close-btn" id="close-menu-btn">X</a>
+      <ul class="a-color">
+         <li><h3>Portfolio</h3><a class="listBtns" href="#profile">></a></li>
+         <li><h3>About</h3><a class="listBtns"  href="#about">></a></li>
+         <li><h3>Contact</h3><a class="listBtns"  href="#contact">></a></li>
```

Comment:
- [ ] When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.

Reply:
The requirement has been fulfilled 👏🏼